### ISSUE TITLE
Unable to Autolink new Native Module Libraries on C# language

### ISSUE THREAD

**Issue by getuliovdmnaweb** [microsoft/react-native-windows#6750]
## Environment
Run the following in your terminal and copy the results here.
1. `npx react-native --version`: 4.13.1
2. `npx react-native info`:
![image](https://user-images.githubusercontent.com/65790521/102223426-75c01700-3ec3-11eb-9ec8-fe7799f11d42.png)

 x64

## Steps To Reproduce

1. Create a New React Native for Windows app following the Get Started doc here: https://microsoft.github.io/react-native-windows/docs/getting-started
2. Create a New Module Library following the Tutorial Here : https://microsoft.github.io/react-native-windows/docs/native-modules-setup
3.  Try to add the new Module Libray as a dependency for you new RNW project.
4. Try to Autolink the new Module Library and run the application.

## Expected Results
Application should build and run with the new Module Library referenced correctly on the new RNW project.

## Snack, code example, screenshot, or link to a repository:
Please provide a Snack (https://snack.expo.io/), a link to a repository on GitHub, or provide a minimal code example that reproduces the problem.

Awesome Project repo :  https://github.com/getuliovdmnaweb/AwesomeProject
MyLibrary Repo:  https://github.com/getuliovdmnaweb/MyLibrary

Error while Building after added MyLibrary as a Reference:
![image](https://user-images.githubusercontent.com/65790521/102226158-d4d35b00-3ec6-11eb-9f99-9eb394b08f74.png)

**Comment by asklar** [comment]
@getuliovdmnaweb the real error is the fact that Microsoft.VCLibs is missing. I've seen this happen recently due to a VS upgrade going wrong. Could you try this and report back?
From an admin cmd prompt:
```bat
msiexec /X {0FAFD370-A0C3-471F-BB08-E2FD0D6430AE}
msiexec /X {580A2D7D-AE4D-4483-8646-CE02F13CBB43}
msiexec /X {1435262F-4DB6-473B-80AF-577F7071EF6E}
msiexec /X {AD98617E-B501-48F0-954E-77F38A5CB424}
msiexec /X {816F9388-9BCF-4F84-8FFC-FDA8B457722C}
```
Then repair Visual Studio 2019, reboot, and try building

**Comment by getuliovdmnaweb** [comment]
I am not sure what these commands should do, but:
![image](https://user-images.githubusercontent.com/65790521/102362060-b470d280-3f92-11eb-8293-7572abebe9d4.png)

Also I user Visual Studio INstaller to Repair Visual Studio 2019 and the same problem is still occurring.

**Comment by getuliovdmnaweb** [comment]
msiexec /X {0FAFD370-A0C3-471F-BB08-E2FD0D6430AE}
This command opens a separate window that the /X seems not even to be a valid command there.
What do you want from that command? What it is suppose to do?

**Comment by asklar** [comment]
Please run them from an elevated cmd prompt, if you are running then in powershell like you are, it will wrongly parse the `{}` as code blocks. These are supposed to uninstall and clean up whatever is leftover from the vclibs install, then it can be repaired by the VS installer

**Comment by getuliovdmnaweb** [comment]
Well, I ran the elevated command promp on Admin Mode, and ran all 5 commands. Only the fourth one uninstalled something. The rest of them did nothing.
I tried to build and now this is what shows.
![image](https://user-images.githubusercontent.com/65790521/102496042-a0da7000-4055-11eb-9a75-927c4b42403b.png)
At least it looks like the other problem went away.

**Comment by asklar** [comment]
Glad to hear that problem is solved :) You should be able to get to a working state by Cleaning the solution then Rebuilding.

**Comment by getuliovdmnaweb** [comment]
Yeah I cleaned and Rebuilded and the same problem is hapenning:
![image](https://user-images.githubusercontent.com/65790521/102501401-2b25d280-405c-11eb-951d-f561e86d2a72.png)

Is there a particular way I should AutoLink the MyLibrary project? It seems not to be working as the documentation outlines.

**Comment by asklar** [comment]
try manually running autolinking from the command line: `npx react-native autolink-windows` then building in VS

**Comment by getuliovdmnaweb** [comment]
I am doing the following Steps:
1 .  Create a new react native for windows project in C# Language (AwesomeProject).
2. Creating a new Native Module Library in C#(MyLibrary).
3. running the AwesomeProject without MyLibrary.
4. using yarn add to add MyLibrary to the project.
5. After That I am trying to build.
6. Build Fails.
Now I did the following:
![image](https://user-images.githubusercontent.com/65790521/102502856-e0a55580-405d-11eb-8e1a-d33d5c45e9f2.png)
And then I cleaned and Rebuild again
![image](https://user-images.githubusercontent.com/65790521/102504931-2c58fe80-4060-11eb-867e-195cff3eeefb.png)
